doc/manual/manual.tex

```diff
@@ -5057,6 +5057,67 @@ \subsubsection{Artificial viscosity smoothing}
 
 
 
+\subsubsection{Tracking finite strain}
+\label{sec:finite-strain}
+\textit{This section was contributed by Juliane Dannberg}
+
+\textit{Notation: Here, we denote the strain rate tensor as $\varepsilon (\mathbf u)$, where 
+$\varepsilon_{ij} = \frac{1}{2}(\frac{\partial u_i}{\partial x_j} + \frac{\partial u_j}{\partial x_i})$ 
```

Comment:
Can you write this as `$varepsilon(\mathbf u)_{ij}` at the top? You probably want `\left(...\right)` around the two fractions to make it look good.